### PR TITLE
[CB-17400] Add SSB-SSE-(UI|WS)(-LB) to knox topology

### PR DESCRIPTION
**Pull request by zsedert**
See detailed description in the commit message.

### CODE REVIEW

File: core-api/src/main/resources/definitions/exposed-services.json [hunk 1]
```diff
@@ -760,6 +760,66 @@
...
+      "knoxUrl": "/ssb-sse-ws/",
+      "ssoSupported": true,
+      "port": 18121,
+      "tlsPort": 18121,
+      "apiOnly": false,
+      "apiIncluded": false,
+      "visibleForDatalake": false,
+      "visibleForDatahub": false,
+      "minVersion": "7.2.16"
+    },
```
Comment (danmagyar) [inline core-api/src/main/resources/definitions/exposed-services.json:792]: These should be mutually exclusive with `SSB-SSC-*` services so please set the max version to `7.2.15` for these two: https://github.com/hortonworks/cloudbreak/blob/520c49bf5814324e21a8709edb8eda5d23ca8a9a/core-api/src/main/resources/definitions/exposed-services.json#L703-L732

File: core-api/src/test/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollectorTest.java [hunk 2]
```diff
@@ -542,6 +610,67 @@ void getNonTLSServicePorts() {
...
+                Map.entry("SCHEMA-REGISTRY-API", 7788),
+                Map.entry("SMM-API", 8585),
+                Map.entry("SMM-UI", 9991),
+                Map.entry("SOLR", 8993),
+                Map.entry("SPARKHISTORYUI", 18088),
+                Map.entry("SPARK3HISTORYUI", 18089),
+                Map.entry("SSB-SSE-UI", 18121),
+                Map.entry("SSB-SSE-WS", 18121),
+                Map.entry("SSB-SSE-UI-LB", 18121),
+                Map.entry("SSB-SSE-WS-LB", 18121),
```
Comment (danmagyar) [inline core-api/src/test/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollectorTest.java:663]: You had set these to `8445` in [exposed-services.json](https://github.com/hortonworks/cloudbreak/pull/12864/files#diff-7ca3fcf95baaf86235de4bb37c5c44707036ce30d8ec2099c64ea31c18a1fe73)

File: core-api/src/test/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollectorTest.java [hunk 3]
```diff
@@ -709,6 +838,67 @@ void getTLSServicePorts() {
...
+                Map.entry("SCHEMA-REGISTRY-API", 7790),
+                Map.entry("SMM-API", 8587),
+                Map.entry("SMM-UI", 9991),
+                Map.entry("SOLR", 8985),
+                Map.entry("SPARKHISTORYUI", 18488),
+                Map.entry("SPARK3HISTORYUI", 18489),
+                Map.entry("SSB-SSE-UI", 18121),
+                Map.entry("SSB-SSE-UI-LB", 18121),
+                Map.entry("SSB-SSE-WS", 18121),
+                Map.entry("SSB-SSE-WS-LB", 18121),
```
Comment (danmagyar) [inline core-api/src/test/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollectorTest.java:891]: You had set these to `8445` in [exposed-services.json](https://github.com/hortonworks/cloudbreak/pull/12864/files#diff-7ca3fcf95baaf86235de4bb37c5c44707036ce30d8ec2099c64ea31c18a1fe73)

**Review by danmagyar — CHANGES_REQUESTED** [review]

File: orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_token.xml.j2 [hunk 4]
```diff
@@ -147,6 +147,22 @@
                  <name>{{ aclsauthz_prefix ~ "ssb-sse-api.acl" }}</name>
                  <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
             </param>
+	    <param>
```
Comment (biharitomi) [inline orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_token.xml.j2:150]: Shouldn't this be behind a condition like in other salt related files of this PR?

**Review by biharitomi — APPROVED** [review]

File: orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_token.xml.j2 (hunk 4 above)
Comment (lacikaaa) [reply orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_token.xml.j2:150]: can we merge this or is this still necessary? @biharitomi

**Pull request merged**